spec/date_parsing_spec.rb

```diff
@@ -423,6 +424,7 @@
           expect(Stanford::Mods::DateParsing.new(example).facet_string_from_date_str).to eq exp
         end
       else
+        expected = expected.to_i.to_s if expected.match(/^\d+$/)
```

Comment:
Is there a context where we're keeping leading 0s, or can we update the `early_numeric_dates` mapping?